acs/acs.go

```diff
@@ -0,0 +1,37 @@
+package acs
+
+type ACS interface {
+	Receive(batch []byte)
+	sendConsensus()
+	sendBatch(id int)
+	sendBinary(id int)
+}
+
+type memACS struct {
+	// TODO : ACS must have rbc instances map
+	// TODO : ACS must have bba instances map
+}
+
+func NewACS() ACS {
+	return &memACS{}
+}
+
+// Receive handles batch which is send from honeybadger.
+func (m *memACS) Receive(batch []byte) {
+
+}
+
+// send sends consensus batch to honeybadger.
+func (m *memACS) sendConsensus() {
+
+}
+
+// sendToRBC sends batch to RBC instance and handles return batch from RBC
```

Comment:
Maybe you had a mistake writing method name. comment and method name is different.